docs/src/tutorials/linear_regression.jl

```diff
@@ -7,12 +7,8 @@ using InteractiveUtils
 # ╔═╡ a4b92179-59f1-4406-a6d8-e2c844f63e95
 # hideall
 let
-    pkg_dir = if "PKGDIR" in keys(ENV)
-        ENV["PKGDIR"]
-    else
-        dirname(dirname(dirname(@__DIR__)))
-    end
-    docs_dir = joinpath(pkg_dir, "docs")
+    docs_dir = dirname(dirname(@__DIR__))
+    pkg_dir = dirname(docs_dir)
```

Comment:
This much simpler version is possible since yesterday (https://github.com/rikhuijzer/PlutoStaticHTML.jl/pull/34).

I'll simplify all notebooks in a next PR.

Reply:
Ok. I'll create an issue.

Reply:
Created #28 for us to follow on.